lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -59,6 +59,7 @@ def build_service_ui
           shell_cmd("yarn install") # TODO: Add --immutable once s390x doesn't change the checksums.
           shell_cmd("yarn run available-languages")
           shell_cmd("yarn run build")
+          shell_cmd("rm -rf node_modules/node-sass/src/binding.cpp")
```

Comment:
Can you add a HACK or TODO comment here so we understand it for the future?  Altnternative, this line can just rm -rf the entire node_modules directory (git clean will kill it anwyay).

Reply:
Done